### PR TITLE
Feat/aktualizr info name only

### PR DESCRIPTION
**Pull request by patrickvacek**
Requested by @jochenschneider.

### CODE REVIEW

File: src/aktualizr_info/aktualizr_info_test.cc [hunk 1]
```diff
@@ -37,13 +37,13 @@ class AktualizrInfoTest : public ::testing::Test {
...
 
-  void SpawnProcess(const char* arg = nullptr) {
+  void SpawnProcess(const std::vector<std::string> args = std::vector<std::string>{}) {
     std::future<std::string> output;
     std::future<std::string> err_output;
     boost::asio::io_service io_service;
     int child_process_exit_code = -1;
 
-    if (arg != nullptr) {
+    for (auto arg : args) {
```
Comment (lbonn) [inline src/aktualizr_info/aktualizr_info_test.cc]: `const auto &arg`?

Comment (patrickvacek) [reply src/aktualizr_info/aktualizr_info_test.cc]: Yep, fixed.

**Comment by patrickvacek** [comment]
@mike-sul any objection to my change to your beautiful new test fixture?

File: src/aktualizr_info/aktualizr_info_test.cc [hunk 2]
```diff
@@ -37,13 +37,13 @@ class AktualizrInfoTest : public ::testing::Test {
...
 
-  void SpawnProcess(const char* arg = nullptr) {
+  void SpawnProcess(const std::vector<std::string> args = std::vector<std::string>{}) {
     std::future<std::string> output;
     std::future<std::string> err_output;
     boost::asio::io_service io_service;
     int child_process_exit_code = -1;
 
-    if (arg != nullptr) {
+    for (const auto& arg : args) {
```
Comment (jochenschneider) [inline src/aktualizr_info/aktualizr_info_test.cc]: Add the whole `args` `vector` at once, something like

`executable_args.insert(std::end(executable_args), std::begin(args), std::end(args));`

File: src/aktualizr_info/main.cc [hunk 3]
```diff
@@ -19,6 +19,7 @@ int main(int argc, char **argv) {
     ("help,h", "print usage")
     ("config,c", po::value<std::vector<boost::filesystem::path> >()->composing(), "configuration file or directory")
     ("loglevel", po::value<int>(), "set log level 0-5 (trace, debug, info, warning, error, fatal)")
+    ("name-only",  "Only output device name (intending for scripting)")
```
Comment (jochenschneider) [inline src/aktualizr_info/main.cc]:
```suggestion
    ("name-only",  "Only output device name (intended for scripting)")
```

File: src/aktualizr_info/aktualizr_info_test.cc (hunk 2 above)
Comment (patrickvacek) [reply src/aktualizr_info/aktualizr_info_test.cc]: Of course! Done.

File: src/aktualizr_info/main.cc (hunk 3 above)
Comment (patrickvacek) [reply src/aktualizr_info/main.cc]: D'oh. Fixed.

**Pull request merged**